pkg/agent/controller/agent.go

```diff
@@ -167,11 +216,18 @@ func (a *Controller) serviceExportToRemoteServiceImport(obj runtime.Object, op s
 	}
 
 	serviceImport.Spec = lighthousev2a1.ServiceImportSpec{
-		Type: lighthousev2a1.SuperclusterIP,
+		Type: svcType,
 	}
+	if svcType == lighthousev2a1.SuperclusterIP {
+		serviceImport.Annotations[clusterIP] = svc.Spec.ClusterIP
```

Comment:
What is this annotation for?

Reply:
This is to handle following use case:
1. There are no healthy pods, remove `ClusterIP` from `ServiceImport`
2. At least one pod is now healthy, add `ClusterIP` back to `ServiceImport`. I am using annotation to avoid doing a read on `Service` to get `ClusterIP`. We can remove this later once we add some caching for objects that we're interested in.